keg_elements/db/mixins.py

```diff
@@ -67,30 +67,116 @@ def from_dict(self, data):
             else:
                 setattr(self, key, value)
 
-    def to_dict(self, exclude=None):
-        if exclude is None:
-            exclude = []
-        insp = sainsp(self)
-        return dict((attr.key, attr.value)
-                    for attr in insp.attrs if attr.key not in exclude)
+    def to_dict(self, exclude=[], hybrids=[]):
```

Comment:
To avoid potential problems in the future, can we use tuple() or frozenset() for the defaults?